packages/babel-standalone/src/preset-stage-2.ts

```diff
@@ -22,6 +22,7 @@ export default (_: any, opts: any = {}) => {
         babelPlugins.proposalRecordAndTuple,
         { syntaxType: recordAndTupleSyntax },
       ],
+      babelPlugins.proposalImportDefer,
```

Comment:
We should enable this plugin only when `opts.modules` is `commonjs` or `cjs`.

Reply:
These presets do not currently have a `modules` option, that's in `preset-2015`. Maybe a `enableProposalImportDefer` option, that we enable from the repl UI when we compile modules? 🤔